### DISCUSSION TITLE
Headings are becoming links

### DISCUSSION THREAD

**Discussion by aetkie** [UNC-Project-Open-AAC/OS-DPI#225] [Help]
The Patterns tutorial is now a live [page](https://github.com/UNC-Project-Open-AAC/OS-DPI/wiki/Patterns-Tutorial) in the Wiki. For some reason, using the headings syntax for markdown (### or ####) is creating links in the text that moves down the document. Can anyone figure out what I am doing wrong? I don't want the links. Just the enlarged headings for Scanning Pattern 1, 2,.. etc.

**Comment by sofia-bg** [comment]
From what I just read it looks like this is how they are auto generated and so that is what is expected to happen. I tried playing around with some syntax but it wasn't working. 

Example: `<span style="color:green;font-weight:700;font-size:20px"> 
markdown color font styles
</span>`

**Comment by gbishop** [comment]
First, that is a feature. They do it so that you can jump to headings inside pages.

Second, and more important, you should not use headings to emphasize text. They should be reserved for structuring the document. Don't think of ###foo as a way to emphasize foo, think of it as a heading named foo.

For emphasis use **bold** or *italic* or both **_italic_**.

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#styling-text
